rift/proto/shard/v2/shard.proto

```
@@ -46,7 +46,7 @@ message Transactions {
 message Transaction {
   string PersonaTag = 1;
   string Namespace = 2;
-  uint64 Nonce = 3;
+  int64 Created = 3;  // microsecond unix timestamp
```

buf

Field "Created" should have a non-empty comment for documentation.

(COMMENT_FIELD)